src/main/java/com/epam/preprod/sirenko/dao/ProductDAO.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.preprod.sirenko.dao;
+
+import com.epam.preprod.sirenko.entity.Clothing;
+import com.epam.preprod.sirenko.entity.Food;
+import com.epam.preprod.sirenko.entity.Product;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+public class ProductDAO {
+	
+	private Map<Integer, Product> productsSetup() {
```

Comment:
We can create initial products during ProductDAO creation

Reply:
done